authzed/validate.sh

```
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+# Validates the canonical Formbricks authorization schema (schema.zed) against
+# the assertion suite in schema-validation.yaml using `zed validate`.
+#
+# Runs fully offline — no SpiceDB server required. Uses a local `zed` binary
+# when available, otherwise the pinned zed container image (the same pin as the
+# authzed-cli service in docker-compose.dev.yml).
+
+set -euo pipefail
+
+readonly SCRIPT_DIR="$(cd -- "$(dirname -- "${BASH_SOURCE[0]}")" && pwd)"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)